### PR TITLE
Add default of current date to new pregnancy create form - issue #279

### LINKED ISSUE #279: New pregnancy create form: Change default to current date
Rolling into one issue since these both came out of @ajohnson052 's pull request in #273  -- 

Here's our form as it stands now -- 

screen shot 2016-05-19 at 10 38 31 am

An open question here: Is there a meaningful difference between initial call date and the day we actually log them in our system? If there is, or if initial call date ever shifts, then we should keep them separate, but otherwise we can probably use the `created_at` timestamp. 

@lwaldsc @NerdyGirl537 can you weigh in on your preferences here from a CM perspective? Should we keep `initial_call_date` as a field or can we infer it based on when we enter them into our system?

### PR DESCRIPTION
**Pull request by Nico-ehs**
Added date default of current date to new pregnancy create form.

Fixes #279

### CODE REVIEW

File: app/controllers/dashboards_controller.rb [hunk 1]
```diff
@@ -11,6 +11,7 @@ def search
     end
     @patient = Patient.new
     @pregnancy = @patient.pregnancies.new
+    @date=Time.now.to_date
```
Comment (Kevin-Wei) [inline app/controllers/dashboards_controller.rb]: Date.today maybe?

Comment (colinxfleming) [inline app/controllers/dashboards_controller.rb]: @Kevin-Wei I think the current Rubocop standard is `Time.zone.today`. I'm inclined to go with that, what do you think @Nico-ehs ?

Comment (colinxfleming) [inline app/controllers/dashboards_controller.rb]: @Nico-ehs what do you think about renaming this variable to something like `@today`? It's not really a generic date but today's date, so I think that might be a little easier to read on the fly. What do you think?

Comment (Kevin-Wei) [inline app/controllers/dashboards_controller.rb]: Rubocop to the rescue

@today sounds pretty good

**Comment by colinxfleming** [comment]
Renamed this PR to reflect what it does in plain language, to make it a touch less confusing. 

@Nico-ehs seeking yr thoughts on the comment thread, but once we have that figured out I think we're all clear!

**Pull request closed without merging**